modules/geo-layers/src/tile-layer/tile-2d-header.js

```diff
@@ -17,12 +18,29 @@ export default class Tile2DHeader {
 
     this.onTileLoad = onTileLoad;
     this.onTileError = onTileError;
+
+    this.transformToWorld = null;
+    this._transformToWorldDataCache = null;
   }
 
   get data() {
     return this._isLoaded ? this.content : this._loader;
   }
 
+  get dataInWorldCoordinates() {
+    if (this.transformToWorld) {
```

Comment:
For me, this is a mistake, if it's called and transformToWorld has not been provided we should just to throw an exception